src/ls/queries.ts

```diff
@@ -3,8 +3,9 @@ import escapeTableName from '../escape-table';
 import { IBaseQueries, ContextValue } from '@sqltools/types';
 
 const describeTable: IBaseQueries['describeTable'] = queryFactory`
-SELECT * FROM
-  ${p => p.isView ? `V_CATALOG.VIEW_COLUMNS` : `V_CATALOG.COLUMNS`}
+SELECT * FROM ${p => p.isView ? `V_CATALOG.VIEW_COLUMNS` : `(SELECT * FROM V_CATALOG.COLUMNS
```

Comment:
I think we are going to run into compatibility issues here with different versions of Vertica. Because V_CATALOG.COLUMNS now has a table_namespace column and V_CATALOG.SYSTEM_COLUMNS does not, this union won't succeed against the soon-to-be released version of Vertica. The new column also isn't at the end of the table, so it's not as clean as just doing a select *, null column_set_using, null is_identity, null table_namespace, it would become:

SELECT * FROM (SELECT * FROM V_CATALOG.COLUMNS UNION SELECT table_id, **null table_namespace**, table_schema, table_name, is_system_table, column_id, column_name, data_type, data_type_id, data_type_length, character_maximum_length, numeric_precision, numeric_scale, datetime_precision, interval_precision, ordinal_position, is_nullable, column_default, null column_set_using, null is_identity FROM V_CATALOG.SYSTEM_COLUMNS) AS U;

I can talk to Upgrade team to see if there is something that we can do to resolve this, but as it is now, with UNION in Vertica there is a good chance we will run into compatibility issues.

Reply:
Another option is to list out everything explicitly except table_namespace from V_CATALOG.COLUMNS which would avoid the compatibility issues in the short term, but then we wouldn't have the namespace context to provide to users

Reply:
Why do we need to include the table_namespace column?   The server implementation Yogesh did should return the _namespace_._schema_ format for the _table_schema_ column.  SQLTools does not know what to do with namespaces and does not need the value.    As long as the schema names are qualified with the namespace names the integration should work.    Siting was going to test with an Eon database with namespaces defined just to make sure.

Reply:
V_CATALOG.COLUMNS and V_CATALOG.VIEW_COLUMNS is not in the list of tables I identified as catalog tables that were used by the drivers at the time we came up with the approach:  https://confluence.verticacorp.com/display/DEV/Namespace+Impact+on+Client+Drivers

Reply:
If we don't want to include table_namespace column at all then we could go with my previous suggestion to list out everything except for table_namespace. I'm not sure what the implications are for that in terms of user experience. If that was removed from this union query what would change?

Reply:
That's the correct behavior.